### PR TITLE
Release notes: Add PR #24866 to release notes

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**
r? @rafaelfranca

### CODE REVIEW

**Comment by vipulnsward** [comment]
Stopped build started by missing `[ci skip]`

File: actionpack/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-*   Add `ActionController#helpers` to get access to the view context in the controller
+*   Add `ActionController#helpers` to get access to the view context at the controller
```
Comment (vipulnsward) [inline actionpack/CHANGELOG.md]: in?

Comment (vipulnsward) [inline actionpack/CHANGELOG.md]: Ah, I din't see level.

**Comment by prathamesh-sonpatki** [comment]
Sorry. Forgot about `[ci skip]` :smile:

File: actionpack/CHANGELOG.md (hunk 1 above)
Comment (prathamesh-sonpatki) [inline actionpack/CHANGELOG.md]: yeah, thats why `at`

**Pull request merged**